### ISSUE TITLE
[feature request]site option to specify external rsyslog server

### ISSUE THREAD

**Issue by samveen** [xcat2/xcat-core#2427]
**TLDR;**
Is it possible to add the option of specifying an external rsyslog server instead of using the xCAT master node/service node? 

**Background:** 
I'm looking to push all my server logs into a log analysis framework and instead of running the bridge app (between the rsyslog and the log analysis framework) on the xCAT management node, I'd like to run it on a seperate dedicated node. 
I am hoping that I can set up a site attribute (something similar to `site.nameservers`) that'll allow me to redirect all logs appropriately.

I have tried to do this logging via setting `network.logservers` for all networks but somehow I am unable to get this to work correctly (and `google` isn't helping).

**Comment by zet809** [comment]
Hi, @hu-weihua and @immarvin , will you pls take a look at this requirement? Thx a lot!

**Comment by immarvin** [comment]
hi @samveen , since all the logs inside the cluster will be forwarded to MN, how about simply redirect all the logs on MN to the syslog server by modifying the syslog,conf on MN? 

is your syslog server reachable from compute nodes or service nodes?

**Comment by samveen** [comment]
@immarvin That is definitely one option. However, given that a lot of services are already running on the xCAT server, I would love the ability to cut it out of the `log` loop completely.

**Comment by immarvin** [comment]
hi @samveen , there are a lot of cluster related logs on MN, for example, log of http server, dhcp/bootp dialogues, xCAT commands log... etc, will you expect these logs to be redirected to the log server so that they can be processed with the logs from the compute nodes and service nodes ?

**Comment by immarvin** [comment]
no response, close this first, please reopen if any update
